### DISCUSSION TITLE
Willing to pay $$$ for the 'dot' double delete bug to be fixed

### DISCUSSION THREAD

**Discussion by lukedupin** [JetBrains/ideavim#816] [General]
Hello,

I've been using Ideavim for >10 years now. Vim has a wonderful feature where you can push 'dot' after input, and it will exactly repeat all your key strokes. This is SUPER handy when you have a fancy action you want to repeat. You can push i, start typing/deleting whatever and finally escape. Now you can push 'n' '.' over and over to repeat that action with searching. MASSIVE time saver.

This feature has had a really unfortunate bug in ideavim, pushing delete removes 2 characters when you hit dot. This small issue makes the feature completely useless.

Is there anyone out there that would fix this on a bounty? I'd be to willing throw in a few hundred $$$ for it to get fixed. Name your price and when I can grab and run the pull request I'll pay ya. If the 'core' devs want money up front to get this fixed, I'd be happy to do that as well. 

The feature is important enough and a big enough pain point, I just need it fixed. Idea vim is a GREAT tool, and the only reason I can effectively use JetBrains. If this one thing were fixed, MY experience would be perfect. Thank you all for the hard work. I hope this is received in a positive light, like a tip after a great meal or something.

**Comment by AlexPl292** [comment]
Hi there. I see that the issue is very annoying for you, sorry for that. I'll try to handle this issue this or next week. [VIM-3055](https://youtrack.jetbrains.com/issue/VIM-3055) (Of course, no bounty is needed).

**Reply by lukedupin** [reply]
Thank you! I'd be happy to donate to help the project in some way.

**Comment by lukedupin** [comment]
OMG!!!! I just got the 2.9.1 update and the double delete has been fixed!!!! Such a massive time savings. Thank you!!!
